Makefile

```diff
@@ -90,7 +90,7 @@ NATIVE_LUAJIT_LIB ?= $(BIN_LUAJIT_DIR)/src/libluajit.a
 LUA_INCLUDE_DIR ?= $(BIN_LUA_DIR)/src
 LUAJIT_INCLUDE_DIR ?= $(BIN_LUAJIT_DIR)/src
 
-COMPILE_ARGS=FENNEL_PATH=src/?.fnl FENNEL_MACRO_PATH=src/?.fnl CC_OPTS=-static
+COMPILE_ARGS=FENNEL_PATH=src/?.fnl FENNEL_MACRO_PATH=src/?.fnl CC_OPTS="-static -I$(LUA_INCLUDE_DIR)"
```

Comment:
This is already set as a CC opt by `--compile-binary`; the final `$(LUA_INCLUDE_DIR)` value is used by Fennel to set the `-I` argument when launching the C compiler here: https://github.com/bakpakin/Fennel/blob/main/src/fennel/binary.fnl#L238

If that's not already working, we should fix it there rather than duplicating the argument.